R/boxr_search.R

```diff
@@ -10,20 +10,21 @@
 #'  for details of the features of the service.
 #'  Some notable details:
 #'  
-#'  * Full-text searching 
-#'    - is available for many source code file types, though not including R at
-#'      the time of writing.
+#'  * Full-text searching is the default
+#'    - available for many source code file types, but not R scripts.
+#'    - by default Box seaches by word/token and uses the `OR` operation e.g.
+#'    `box_search("this that")` is equivilant to `box_search("this OR that")` 
 #'       
-#'  * Boolean operators are supported 
-#'    - such as `and`, `or`, and `not` (upper or lower case).
+#'  * Reserved words for boolean operations
+#'    - `AND`, `OR`, and `NOT` (uppercase only) are interpreted as special context e.g.
+#'    `box_search("NOT this")`, `box_search("this AND that")`
 #'       
-#'  * Phrases can be searched
-#'    - by surrounding them with "quotation marks" in the query string e.g. 
-#'      `box_search("match similar phrases")`, `box_search("\"match this phrase exactly\"")`
+#'  * Exact phrases can be matched
+#'    - by surrounding them with double quotation marks e.g.
+#'    `box_search('"this exact phrase"')` or `box_search("\"this exact phrase\"")` 
```

Comment:
@nathancday, `box_search('"this exact phrase"')` does not perform an exact search. I tested it on my computer, but you can verify at your end. So, adding this example will be misleading.

Reply:
Looks like we are getting different results. 🤷

Reply:
Huh, that's interesting! Then leave the example there. :)